config/confd/conf.d/logentries.conf.toml

```diff
@@ -1,6 +0,0 @@
-[template]
```

Comment:
These files may actually be needed as we're using logentries in balenaCloud. With this change, how will the registry publish logs?

Reply:
so after this change the registry will log to stdout... do we have something like loki setup for balenacloud services?

Reply:
We currently stream service logs to logentries -- I'm not sure how many (if any) services switched over to Loki.

For open-balena this is no problem at all, stdout works just fine there.